[2024-07-22T02:35:30.305+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: example_dynamic_task_mapping.add_one scheduled__2022-12-02T00:00:00+00:00 map_index=1 [queued]>
[2024-07-22T02:35:30.316+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: example_dynamic_task_mapping.add_one scheduled__2022-12-02T00:00:00+00:00 map_index=1 [queued]>
[2024-07-22T02:35:30.316+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-07-22T02:35:30.318+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2024-07-22T02:35:30.319+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-07-22T02:35:30.460+0000] {taskinstance.py:1300} INFO - Executing <Mapped(_PythonDecoratedOperator): add_one> on 2022-12-02 00:00:00+00:00
[2024-07-22T02:35:30.468+0000] {standard_task_runner.py:55} INFO - Started process 3078 to run task
[2024-07-22T02:35:30.490+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'example_dynamic_task_mapping', 'add_one', 'scheduled__2022-12-02T00:00:00+00:00', '--job-id', '1123', '--raw', '--subdir', '/home/***/.local/lib/python3.7/site-packages/***/example_dags/example_dynamic_task_mapping.py', '--cfg-path', '/tmp/tmpd5y3y364', '--map-index', '1']
[2024-07-22T02:35:30.493+0000] {standard_task_runner.py:83} INFO - Job 1123: Subtask add_one
[2024-07-22T02:35:30.575+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 1123 for task add_one ([Errno 2] No such file or directory: '/opt/airflow/logs/dag_id=example_dynamic_task_mapping/run_id=scheduled__2022-12-02T00:00:00+00:00/task_id=add_one/map_index=1/attempt=1.log'; 3078)
[2024-07-22T02:35:30.590+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2024-07-22T02:35:30.612+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
